src/main/webapp/i18n/de/statistics.json

```
@@ -19,7 +20,9 @@
         "exam_registrations": "Klausuranmeldungen",
         "active_tutors": "Aktive Tutoren",
         "created_results": "Erstellte Resultate",
-        "created_feedbacks": "Erstellte Rückmeldungen"
+        "created_feedbacks": "Erstellte Rückmeldungen",
```

```suggestion
        "created_feedbacks": "Erstellte Feedbacks",
```
In all other places the German translations kept Feedback.

---

👍